<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QLabel" name="label_1">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
  border:1px solid;
border-radius : 10px;
 background-color : white;
color:black;

}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
  border:1px solid;
border-radius : 10px;
 background-color : white;
color:black;

}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>c</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QPushButton" name="pushButton_25">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border : 1px solid black;
background-color : #9AECDB;
color:black;
border:1px solid black;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Advance</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_30">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>Theme</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_19">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_31">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>1/x</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_seven">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_eight">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_nine">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_5">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButton_26">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_four">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_five">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_six">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_9">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="pushButton_27">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_one">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton_two">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton_three">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="pushButton_13">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="pushButton_28">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>√</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_zero">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border : 1px solid;

border-radius : 10px;
}

QPushButton:pressed
{
 background-color:#b3ffff;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButton_17">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="pushButton_20">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

border : 1px solid black;
background-color : #9AECDB;
color:black;
border:1px solid black;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="pushButton_29">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
  border:1px solid;
border-radius : 10px;
 background-color : #bdc3c7;
color:black;

}

</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
